testing/mbuffer/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Katie Holly <holly@fuslvz.ws>
+# Maintainer: Katie Holly <holly@fuslvz.ws>
+pkgname=mbuffer
+pkgver=20171011
+pkgrel=0
+pkgdesc="mbuffer buffers I/O operations and displays the throughput rate"
+url="http://www.maier-komor.de/mbuffer.html"
+arch="all"
+license="GPL-3.0"
+depends=""
+makedepends="linux-headers libressl-dev"
+install=""
+subpackages="$pkgname-doc"
+source="
+	http://www.maier-komor.de/software/mbuffer/mbuffer-$pkgver.tgz
+	"
+
+builddir="$srcdir"/mbuffer-$pkgver
+
+prepare() {
+	cd "$builddir"
+	default_prepare
+}
```

Comment:
The entire `prepare()` function can be removed since the default is to run `default_prepare`